### ISSUE TITLE
packaged Windows application starts then crashes, debug version runs, how to debug .exe

### ISSUE THREAD

**Issue by billacw** [wailsapp/wails#325]
**Description**

Go code and React frontend works fine in debug/dev mode. Wails serve starts, npm run start both work. After packaging the application into a Windows .exe and running it, the Webview flashes and ends. How can I debug the .exe?

I ran the executable file and got this error from Visual Studio
API server listening at: 127.0.0.1:29900
could not launch process: decoding dwarf section info at offset 0x0: too short
Process exiting with code: 1

**To Reproduce**
Steps to reproduce the behaviour:

**Expected behaviour**

**Screenshots**
Go Environment
![go env](https://user-images.githubusercontent.com/58347572/72003589-8a4d5700-31fe-11ea-8862-3673445dae89.png)

**System Details**

| Name   | Value |
| ----- | ----- |
| Wails Version | v1.0.2-pre2 |
| Go Version    | go1.13 |
| Platform      | windows |
| Arch          | amd64 |
| GO111MODULE   | (Not Set) |
| GCC           | 9.2.0 |
| Npm           | 6.12.1 |
| Node          | v12.13.1
 |

**Additional context**
console output during running normally

 wails bridge  Connected to backend 
init.js:26  wails bridge  Loaded Wails Runtime 
init.js:26  wails bridge  Wails Ready 
init.js:26  wails bridge  Notifying application 
index.js:1406 Warning: Invalid DOM property `for`. Did you mean `htmlFor`?
    in label
    in StyledComponent
    in styled.label (at InputFileAttach.js:33)
    in div
    in StyledComponent
    in styled.div (at InputFileAttach.js:32)
    in InputFileAttach (at MainStep.js:27)
    in div (at MainStep.js:19)
    in MainStep (at src/index.js:12)
console.<computed> @ index.js:1406

RMConnStatus.js:11 Got RM token
DPConnStatus.js:13 connected to DocParser
index.js:1406 Warning: Each child in a list should have a unique "key" prop.

Check the render method of `ParserList`. See https://fb.me/react-warning-keys for more information.
    in option (at ParserList.js:49)
    in ParserList (at MainStep.js:29)
    in div (at MainStep.js:19)
    in MainStep (at src/index.js:12)
console.<computed> @ index.js:1406

ParserList.js:16 (23) [{…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}]
ParserList.js:20 first parser return val: Parser id is now: gokavmvilpjy

**Comment by leaanthony** [comment]
Based on a [quick google search](https://www.google.com/search?q=decoding+dwarf+section+info+at+offset+0x0%3A+too+short), this sounds like an environment issue. Does anything in those links apply to your setup?

**Comment by billacw** [comment]
I think it was an environment issue. I restarted the app by creating a new Wails project with init in a separate directory. I carefully added in only the node modules I needed. I made sure I could build an app at each step and have it run. I slowly added in my go code and bit by bit the React code. I put in global onerror in the html body. Never found any issue that caused any problem in the new directory/app. The only change made was to use htmlFor= in a <label> instead of for=. That removed a warning I got in console.log in Chrome. Now the completely rebuilt app in a clean environment works in debug and as a package.
